nd4j/nd4j-tensorflow/src/main/java/org/nd4j/tensorflow/conversion/ProtoBufToFlatBufConversion.java

```diff
@@ -0,0 +1,160 @@
+package org.nd4j.tensorflow.conversion;
```

Comment:
Only only last thing I think, we'd need a copyright header from Skymind at the top here.

Reply:
Done